recipes/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "related" %}
```

Comment:
You need to move this file from `recipes/meta.yaml` to `recipes/related/meta.yaml`

Reply:
+1